.github/workflows/test-setup-kind.yaml

```diff
@@ -33,6 +33,7 @@ jobs:
       uses: ./setup-kind
       with:
         k8s-version: ${{ matrix.k8s-version }}
+        cluster-suffix: c${{ github.run_id }}.local
```

Comment:
maybe add another test without the cluster-suffix

Reply:
done